asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -189,11 +204,19 @@ def from_smiles(cls, smiles: str, **kwargs) -> "Ligand":
     @property
     def smiles(self) -> str:
         """
-        Get the SMILES string for the ligand
+        Get the canonical isomeric SMILES string for the ligand
         """
         mol = self.to_oemol()
         return oemol_to_smiles(mol)
 
+    @property
+    def non_iso_smiles(self) -> str:
```

Comment:
Perhaps `canonical_smiles` rather than not iso?

Reply:
the thing is that they are both "canonical", the only thing that makes this one different is the fact that this one is not isomeric

Reply:
Fair enough.